core-bundle/src/Resources/contao/dca/tl_form_field.php

```diff
@@ -575,15 +660,17 @@ public function optionImportWizard()
 	 */
 	public function getFields()
 	{
-		$arrFields = $GLOBALS['TL_FFL'];
+		$fields = array();
 
-		// Add the translation
-		foreach (array_keys($arrFields) as $key)
+		foreach ($GLOBALS['TL_FFL'] as $k=>$v)
 		{
-			$arrFields[$key] = $GLOBALS['TL_LANG']['FFL'][$key][0];
```

Comment:
Did you intentionally change the field array definition? I didn't see a new `reference` property?

Reply:
Not sure what you mean… 🤷‍♂

Reply:
The method now only returns the field types, where it previously returned the field type as key and the label as value. Is the select menu still translated, or should you also add a `reference` DCA property to it?

(also, this _might_ be considered a BC break, or at least it seems unnecessary…)

Reply:
Yes, there is a `reference` entry in the DCA. It has been there all the time, that is why it does not show up in the diff.